src/Simplex/Chat/Types.hs

```diff
@@ -335,9 +335,8 @@ data UserContactRequest = UserContactRequest
     profile :: Profile,
     createdAt :: UTCTime,
     updatedAt :: UTCTime,
-    xContactId :: Maybe XContactId
-    -- TODO PQ save pqSupport from REQ to database
-    -- pqSupport :: PQSupport
+    xContactId :: Maybe XContactId,
+    pqSupport :: Maybe PQSupport
```

Comment:
No need for it to be Maybe here

Reply:
no need for it to be NOT NULL in database either, but whatever